bits/BIT-0005-Start-Call.md

```diff
@@ -0,0 +1,50 @@
+# BIT-0005: dTAO Subnet Start Call Function
+
+- **BIT Number:** 0005
+- **Title:** dTAO Subnet Start Call
+- **Author(s):** CapriciousSage, Rhef, Vune
+- **Status:** Final
+- **Type:** Core
+- **Created:** 28/02/2025
+- **Updated:** 05/03/2025
+
+## Abstract
+
+Introduce a "Start Call" function to all new subnets and change the default registration toggle to `False`. Once a new subnet is registered, UID registrations would be disabled by default and subnet epoch/emissions do not start (pool locked, inflation paused) until the Subnet Owner executes both the start call function (which unlocks the pool, starts inflation, and - optionally - enables registrations).
+
+## Motivation
+
+Pre-dtao, new subnets experienced a 7-day immunity period, which allowed a subnet owner to verify the discord channel, publish their repo, onboard valis and miners, and 'get things rolling' - all without pressure of deregistration and without any real money on the table.
+
+Meanwhile, dTAO creates a free-for-all 'rush' that incentivizes bogus validator or mining code to try and establish early vtrust and child hotkey delegations, undermining new subnets right out of the gate. This not only muddies the early token supply (causing undue market damage) but also undermines the subnet owner's attempts to launch the subnet correctly.
+
+By allowing subnet owners to control when the subnet "starts", Subnet owners get a chance to establish themselves without being pushed around in their own network, and network participants get time to evaluate "do I want to mine, validate or buy this token" (minimising the chances of getting rekt on a rugnet/ponzi). It also subnet ensures registration costs remain at steady prices (maximising burn for the ecosystem) by making opportunity registrations more appealing.
+
+## Specification
+
+When new subnets are registered, they are effectively dormant. No tao/dtao added and no trading possible, no dtao Token Emissions, and no UID registrations.
+
+When the subnet owner is ready to initialise the subnet, they execute the "start call" function (similar to Parachain), which unlocks the LP (allowing trading), and starts epoch/emissions, with the option to also enables UID registrations. We also propose that "start call" cannot be triggered for at least 50,400 blocks (7 days) after the subnet has been registered to minimise the chances of haphazard launches or unfounded retail FOMO.  
+
+NOTE: Subnet owners should be able to enable UID registrations prior to/without executing the "start call" to give validators and miners an opportunity to register and deploy "real code" before the wheels start turning, however there would be a function to optionally enable both things at once depending on the flag set.
```

Comment:
I wonder if allowing registrations may encourage increasing the pool ratio (increases TAO, decreases Alpha) via registration recycling when no staking can happen.